### PR TITLE
GT-1323 Add new tutorial for english 

### PR DESCRIPTION
**Pull request by levieggertcru**
- Adding new tutorial assets for when device is in english language
- Keeping original tutorial when device is not in english

### CODE REVIEW

File: godtools/App/Features/Tutorial/TutorialFlow.swift [hunk 1]
```diff
@@ -24,6 +26,7 @@ class TutorialFlow: Flow {
         
         self.flowDelegate = flowDelegate
         self.appDiContainer = appDiContainer
+        self.flowDiContainer = TutorialFlowDiContainer(appDiContainer: appDiContainer)
```
[the remark this answers is not in the document]
Comment (levieggertcru) [reply godtools/App/Features/Tutorial/TutorialFlow.swift:29]: Hey @reldredge71 basically I'm testing a slightly different structure using clean architecture (adding domain layer) with this specific feature with the possibility of moving our other features to this format.  
The main idea is to
1. Move specific feature dependencies into each feature folder and use the flowDiContainer to fetch any useCases or viewModel dependencies specific to that feature.  This will also help reduce clutter in the AppDiContainer and make it easier to see which features depend on what.  AppDiContainer would still contain any shared or core dependencies that are used across features.
2. Move business logic out of the viewModel and into useCases to help with re-using useCases across viewModels and to reduce viewModel responsibilities.

I'm glad you noticed that and asked.  Also, having a feature specific dependency container (TutorialFlowDiContainer) allows for the possibility of injecting containers into the feature that may point to different data layers and useCase implementations.  In our case we're instantiating TutorialFlowDiContainer directly, but it opens up this possibility.

[the remark this answers is not in the document]
Comment (levieggertcru) [reply godtools/App/Features/Tutorial/TutorialFlow.swift:29]: I'll try to find an example of where I've seen this setup used.  I believe GodTools Android uses something similar. @gyasistory  would know.

File: godtools/App/Features/Tutorial/Presentation/Tutorial/TutorialView.swift [hunk 2]
```diff
@@ -7,10 +7,9 @@
 //
 
 import UIKit
-import youtube_ios_player_helper
 
+// TODO: Look into using TutorialPagerView instead. ~Levi
```
Comment (reldredge71) [inline godtools/App/Features/Tutorial/Presentation/Tutorial/TutorialView.swift:11]: Would you like to do this in another PR?  I'm hoping it won't be too difficult to apply TutorialPagerView here.

Comment (levieggertcru) [reply godtools/App/Features/Tutorial/Presentation/Tutorial/TutorialView.swift:11]: Hey @reldredge71 I'm going to create a separate Jira Task for that.  There is another place that needs the TutorialPageView as well which can be part of that task.

**Review by reldredge71 — DISMISSED** [review]
